Tests/x10/ops_test.swift

```diff
@@ -1273,6 +1273,47 @@ final class TensorTests: XCTestCase {
       }
     }
   }
+    
```

Comment:
Remove the space at the beginning of the line.

Reply:
Done.